action.yml

```
@@ -232,15 +232,20 @@ runs:
       env:
         GITHUB_TOKEN: "${{ inputs.token }}"
       run: |
-        output_file=$(atmos docs generate ${{ inputs.target }} 2>&1 | sed -n -e  "s/^.*output=//p")
-        # Get relative path from absolute
-        output_file=$(realpath -s --relative-to="$PWD" "$output_file")
-        # Ignore changes if they are only whitespace
-        if ! git diff --quiet ${output_file} && git diff --ignore-all-space --ignore-blank-lines --quiet ${output_file}; then
-          git restore ${output_file}
-          echo Ignoring whitespace-only changes in generated file
-        fi
-        echo "file=${output_file}" >> $GITHUB_OUTPUT
+        set +e
+        atmos ${{ inputs.command }}
```

this could be a custom command that gives bash access to the runners

---

if we don't care about that then we could just simply do 
```suggestion
        ${{ inputs.command }}
```

and default to `atmos docs generate readme`